extensions/blocks/contact-form/components/jetpack-field-checkbox.js

```diff
@@ -10,9 +10,10 @@ import { withInstanceId } from '@wordpress/compose';
  * Internal dependencies
  */
 import JetpackFieldLabel from './jetpack-field-label';
+import JetpackFieldWidth from './jetpack-field-width';
 
 function JetpackFieldCheckbox( props ) {
-	const { instanceId, required, label, setAttributes, isSelected, defaultValue } = props;
+	const { instanceId, required, label, setAttributes, fieldWidth, defaultValue } = props;
```

Comment:
I think the prop could be just `width` since the field is implicit (also consistent with `label` etc).

Reply:
Changed all instances to `width`. I had this originally but wondered if it would cause any conflicts.